### DISCUSSION TITLE
Requesting test funds

### DISCUSSION THREAD

**Discussion by FaradayNodes** [entropyxyz/community#122] [Get Test Funds] [answered]
### Address

5EqUREuFWVDD16CUKvverPmgBAezFMexHKFbjVbponSq9ioi

**Comment by johnnymatthews** [comment, marked as answer]
Sent in block [66,204](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftestnet.entropy.xyz#/explorer/query/0xc6729f171fef88f28f8d4fee36399b0dd728fb9716f93a73a2faba243dd4c58e)
